Documentation/configuration.md

```diff
@@ -13,6 +13,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 ##### -name
 + Human-readable name for this member.
 + default: "default"
++ This value is referenced as one of this node's own entries listed in the `-initial-cluster` flag (Ex: `default=http://localhost:2380`). This needs to match the key used in the flag if you're using [static boostrapping](clustering.md#static).
```

Comment:
This value is referenced as this node's own entries listed in the `-initial-cluster` flag (Ex: `default=http://localhost:2380` or `default=http://localhost:2380,default=http://localhost:7001`).